gitreceive/receiver/flynn-receive.go

```diff
@@ -199,13 +199,10 @@ Options:
 		if err := client.PutFormation(formation); err != nil {
 			log.Fatalln("Error putting formation:", err)
 		}
-		fmt.Println("=====> Waiting for web job to start...")
 
 		err = watcher.WaitFor(ct.JobEvents{"web": {"up": 1}}, scaleTimeout, func(e *ct.Job) error {
-			switch e.State {
-			case "up":
-				fmt.Println("=====> Default web formation scaled to 1")
-			case "down", "crashed":
+			fmt.Printf("%s ==> %s %s %s\n", time.Now().Format("15:04:05.000"), e.Type, e.ID, e.State)
+			if e.State == "down" || e.State == "crashed" {
```

Comment:
This will only print on the first deploy, which is incorrect. I think the timestamp should be at the end in parens and the arrows should be the same length for symmetry.